src/services/reviews.js

```diff
@@ -173,16 +175,15 @@ module.exports = class reviewsHelper {
 
 			// Update the status in the reviews table
 			// Update the 'last_reviewed_on' field in the resources table
-			await Promise.all([
-				reviewsQueries.update(
-					{ id: review.id, organization_id: review.organization_id },
-					{ status: common.REVIEW_STATUS_INPROGRESS }
-				),
-				resourceQueries.updateOne(
-					{ organization_id: resource.organization_id, id: resourceId },
-					{ status: common.RESOURCE_STATUS_IN_REVIEW }
-				),
-			])
+			await reviewsQueries.update(
+				{ id: review.id, organization_id: review.organization_id },
+				{ status: common.REVIEW_STATUS_INPROGRESS }
+			)
+			const resourceStatus = await resourceService.finalResourceStatus(resourceId, resource.organization_id)
```

Comment:
@priyanka-TL why we need final status call, it will always be IN_PROGRESS if its submitted for review, else don't update something like that, check if we can avoid this call

Reply:
@aks30 as we discussed in morning, consider there is 2 reviewer, and the reviewing parallel, one reviewer is alreday requested_for_change and then the resource status will be updated a requested_for_change. When the second person is start the review the resource status should not set as INPROGRESS hence the priority will be requested_for_change

Reply:
Please connect with @adithyadinesh0412 i explained how we can potentially do it